packages/taquito-utils/test/format.spec.ts

```diff
@@ -21,4 +22,24 @@ describe('Format', () => {
   it('Should convert tz to mutez', () => {
     expect(format('tz', 'mutez', 1)).toEqual(new BigNumber(1000000));
   });
+
+  it('Should be valid bytes tezostaquito example', () => {
```

Comment:
could you update the description to be more specific. This tests a specific part in the doc on signing bytes through a wallet. We should make that reflect the test name. It also lives in a completely separate part of the codebase.

You could give a link in the form of a comment to the section in the doc. 

It's also prob a good idea to make a new describe block since this doesn't seem like it's the same set of cases.